wme-sitebuilder/Container.php

```diff
@@ -97,7 +97,11 @@ public function config() {
 					$app->make( Contracts\ManagesDomain::class )
 				);
 			},
-			Wizards\LookAndFeel::class            => null,
+			Wizards\LookAndFeel::class            => function ( $app ) {
+				return new Wizards\LookAndFeel(
+					$app->make( Modules\StoreDetails::class )
```

Comment:
I'm still a bit uncomfortable with explicitly giving a wizard a module. Modules should include wizards and wizards should be able to pass any information they need.

That being said, I know we don't really have a "Manager" that would be able to provide the next module in the series. 

Could we instead replace the implicit call to `[Wizards/LookAndFeel.php:72] $this->store_details->getPageUrl(),` with an `apply_filters` call that could then provide the link from within the the integration (ex: [Nexcess\MAPPS\Integrations\StoreBuilder](https://github.com/liquidweb/nexcess-mapps/blob/a8e45e2c31f330c561cbf4aa958afbb37f61736c/nexcess-mapps/Integrations/StoreBuilder.php#L147))?

```php
// Two potential options for the filter definition
'storeDetailsURL' => apply_filters( 'wme_sitebuilder_redirect_url', '', $this->admin_page_slug, $this->wizard_slug ),
'storeDetailsURL' => apply_filters( "wme_sitebuilder_redirect_url_{$this->admin_page_slug}_{$this->wizard_slug}", '' ),
```

```php
// Example of how to handle the integration from within Nexcess\MAPPS\Integrations\StoreBuilder.
add_filter( 'wme_sitebuilder_redirect_url_sitebuilder_look-and-feel', function( $url ) {
	$store_details = SitebuilderContainer::getInstance()->get( Modules\StoreDetails::class );
	return $store_details->getPageUrl() ;
});
```

I know it's more complex, but it allows us to keep a clean delineation between Modules and Wizards where class composition only goes in one direction. The first time we open it up to Modules and Wizards being intermingled, it's going to set a precedent that will be difficult to unwind.

Ideally we would register a NavigationManager which would keep track of the order in which modules are registered and if completing one module should redirect to the start of the next. Until we have a use case for something of that complexity, applying filters would be a simple way of allowing the integration to define where to navigate next.